packages/react/src/components/FileUploader/index.js

```diff
@@ -5,24 +5,9 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import * as FeatureFlags from '@carbon/feature-flags';
-import Filename from './Filename';
-import FileUploader from './FileUploader';
-import FileUploaderSkeleton from './FileUploader.Skeleton';
-import FileUploaderButton from './FileUploaderButton';
-import FileUploaderDropContainer from './FileUploaderDropContainer';
-import FileUploaderItem from './FileUploaderItem';
-import { createClassWrapper } from '../../internal/createClassWrapper';
-
-export {
-  Filename,
-  FileUploader,
-  FileUploaderSkeleton,
-  FileUploaderButton,
-  FileUploaderDropContainer,
-  FileUploaderItem,
-};
-
-export default FeatureFlags.enabled('enable-v11-release')
-  ? createClassWrapper(FileUploader)
-  : FileUploader;
+export default from './Fileuploader';
```

Comment:
I think the typo here is causing the CI to fail
```suggestion
export default from './FileUploader';
```

Reply:
@abbeyhrt thank you ! I was going crazy trying to figure it out lol